wagtail/admin/views/pages/delete.py

```diff
@@ -50,5 +78,10 @@ def delete(request, page_id):
             "page": page,
             "descendant_count": page.get_descendant_count(),
             "next": next_url,
+            "translation_count": len(pages_to_delete),
```

Comment:
as in the other case. The confirmation message specifically refers to translations. An alias page is not a translation but a direct copy which gets removed.

as such this should be 

```python
"translation_count": pages_to_delete.filter(alias_of__isnull=True).exclude(page_to_delete).count(),
```